test/passing/refs.default/sequence-preserve.ml.ref

```diff
@@ -112,8 +112,8 @@ let _ =
 let foo x y =
   do_some_setup x;
   let* () = do_some_setup y in
-
   (* Empty line after letop *)
+
```

Comment:
This change is totally fine. Comments are parsed separately associated to closest locations collected from the AST using the `Ast_mapper`. This comment is closer to where it was in the original source (just after the `in` and not before the next expression) and that's what we want.